src/App.tsx

```diff
@@ -1,14 +1,33 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(false);
+  const [initialTodos, setInitialTodos] = useState<Todo[] | null>([]);
```

Comment:
```suggestion
  const [initialTodos, setInitialTodos] = useState<Todo[]>([]);
```
initial todos never wil be null. it is avoid to check todos in al places.